packages/product-react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,92 @@
+import clsx from 'clsx';
+import React, { FC, CSSProperties } from 'react';
+import RTLWrapper from '../Common/RTLWrapper';
+import './Card.scss';
+
+const LINKWrapper: React.FC<{ props?: ILink }> = ({ children, props }) =>
+  props ? <a {...props}>{children}</a> : <>{children}</>;
+
+export interface ILink extends React.HTMLProps<HTMLAnchorElement> {
+  download?: any;
+  href?: string;
+  hrefLang?: string;
+  media?: string;
+  rel?: string;
+  target?: string;
+  type?: string;
+  style?: CSSProperties;
+  className?: string;
+}
+
+export interface ICardProps extends React.HTMLProps<HTMLDivElement> {
+  image?: React.ReactNode;
+  header?: string;
+  content?: string;
+  row?: boolean;
+  compact?: boolean;
+  inactive?: boolean;
+  rtl?: boolean;
+  linkProps?: ILink;
+}
+
+export const Card: FC<ICardProps> = ({
+  image,
+  header,
+  content,
+  row,
+  compact,
+  inactive,
+  children,
+  className,
+  rtl,
+  linkProps,
+  ...rest
+}) => {
+   return (
+    <LINKWrapper props={linkProps}>
```

Comment:
Hmmm these names are interesting! I think regular Capitalized camelcase should be ok instead.